normalizer/normalizer.bal

```diff
@@ -0,0 +1,45 @@
+import ballerina/http;
+
+type ZendeskResponse record {
+    record {|
+        string url;
+        int id;
+        string subject;
+    |} ticket;
+};
+
+service /api/v1 on new http:Listener(8080) {
+    
+    resource function post ticket(@http:Payload json|xml request) returns xml|json|error {
+        json zendeskRequest = {};
+        if request is json {
+            zendeskRequest = {
+                ticket: {
+                    subject: check request.subject, 
+                    comment: {
+                        body: check request.comment
+                    }
+                }
+            };
+        }
+        if request is xml {
+            zendeskRequest = {
+                ticket: {
+                    subject: (request/<subject>).data(), 
+                    comment: {
+                        body: (request/<comment>).data()
+                    }
+                }
+            };
+        }
+        if zendeskRequest != {} {
+            http:Client zendeskClient = check new ("http://api.zendesk.com.balmock.io");
+            ZendeskResponse result = check zendeskClient->/api/v2/tickets.post(zendeskRequest);
+            return {
+                id: result.ticket.id,
+                subject: result.ticket.subject,
+                url: result.ticket.url
+            };
```

Comment:
Let's return the URL or id to simplify.

Reply:
returned a tuple of [int, string, string].